cbmackintosh/overlook
# Feature/design and accessibility

**Clearly and concisely describe the feature.**
- Adds stylization and refactors CSS into SASS with some mixins to reduce repetitious code
- Adds accessibility features to charts and resolves errors on accessibility audits
- Responsiveness is slightly improved although could still use some work on the manager interface

**Is this a feature or refactor?**
- Refactor

**Problems Encountered & Resulting Solutions**
- None

**Where should the reviewer start?**
- a CSS partial was created for reusable code that can be imported into the CSS files for each interface
- CSS was udpated and refactored in all stylesheets
- Some accessbility features were added to the chart compilers in manager-interface.js

**How should this be tested?**
- Download the application, test with a screen-reader, test with tabbing. Conduct accessibility audits with WAVE and Lighthouse to verify scores are adequate.

**Steps To Take**
- Look into network error handling before final project submission.
- Review responsiveness on manager interface
